modin/backends/base/query_compiler.py

```diff
@@ -1488,6 +1495,7 @@ def pivot_table(
         dropna,
         margins_name,
         observed,
+        sort,
```

Comment:
`sort` is a new parameter for `pivot_table` introduced in pandas 1.3.0, so we have to pass it to the backend as well
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.pivot_table.html